zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/VFSNotebookRepo.java

```diff
@@ -134,10 +137,23 @@ public Note get(String noteId, String notePath, AuthenticationInfo subject)
         NameScope.DESCENDENT);
     String json = IOUtils.toString(noteFile.getContent().getInputStream(),
         conf.getString(ConfVars.ZEPPELIN_ENCODING));
-    Note note = noteParser.fromJson(noteId, json);
-    // setPath here just for testing, because actually NoteManager will setPath
-    note.setPath(notePath);
-    return note;
+
+    try {
+      Note note = noteParser.fromJson(noteId, json);
+      // setPath here just for testing, because actually NoteManager will setPath
+      note.setPath(notePath);
+      return note;
+    } catch (CorruptedNoteException e) {
+      String errorMessage = String.format(
```

Comment:
Please do not use newlines in short messages. If the error is logged, this makes parsing unnecessarily difficult.

Reply:
@Reamer 
I've removed the newlines. I didn't consider the logging. To me, the popup seems a bit harder to read now. But ensuring operational convenience could be more important.